lib/gui/app.js

```diff
@@ -214,6 +215,24 @@ app.run(($timeout) => {
   })
 
   driveScanner.on('error', (error) => {
+    if (error.message === 'LIBUSB_ERROR_ACCESS') {
+      return permissions.isElevated().then((elevated) => {
+        if (!elevated) {
+          // Stop all adapters but notably usbboot, remove usbboot, and start the remaining adapters
```

Comment:
@jhermsmeier Does this usage-pattern suggest there should be a "better" way to dynamically add and remove adapters?

Reply:
Yeah, the current implementation isn't complete or anything, just what we needed to make it work, and get closer to what we want in terms of the adapters.

Reply:
@Shou does this mean anyone starting Etcher on Linux will get this warning at startup?

Reply:
Only if they have a usbboot device plugged in.

Reply:
...and even then, if they have 'enough' permissions to open USB devices without being root, AIUI they won't see this message either.

Reply:
@lurch just realized that we do have a method for this: `scanner.unsubsribe(adapter)`

Reply:
@jhermsmeier You meant `unsubscribe`, right? :rofl:

Reply:
Oh jesus... man, how do I keep mucking that up? 😆